### ISSUE TITLE
Accessibility issues in forms

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/derrida-django#228]
- [x] missing alt tags for drop-down images
- [x] missing labels 
- [x] duplicate labels in your search box at top right 
- [x] duplicate labels in filters

See wave report here: http://wave.webaim.org/report#/https://derridas-margins.princeton.edu/references/

**Comment by thatbudakguy** [comment]
Looks like these fields are still missing form labels, according to WAVE:
- [ ] original publication year
- [ ] edition year
- [ ] printing year

There are three other "missing labels" and an "empty button", but I can't see them on the page and WAVE won't tell me what/where they are...

Insufficient contrast in a few places, as well:
- [ ] version number
![screen shot 2018-10-08 at 10 40 15 am](https://user-images.githubusercontent.com/4924494/46615612-9ff93b80-cae6-11e8-9d77-b1340884cfa5.png)
- [ ] labels
![screen shot 2018-10-08 at 10 39 48 am](https://user-images.githubusercontent.com/4924494/46615632-b30c0b80-cae6-11e8-8235-c0ce634ac162.png)
- [ ] pagination
![screen shot 2018-10-08 at 10 40 12 am](https://user-images.githubusercontent.com/4924494/46615648-bdc6a080-cae6-11e8-81f0-91d51aff0206.png)

**Comment by rlskoeser** [comment]
Added `aria-label` attributes to hidden inputs associated with range filters. Adjusted colors for software version label and result count to address contrast; not changing theme-specific colors. 

Checked with WAVE on the test site to confirm the fixes.
